admin_solutions/user_role_mgmt.adoc

```diff
@@ -0,0 +1,542 @@
++
+----
+$ oc create -f resource-quota.yaml -n awesomeproject
+----
++
+. After the quota has been in effect for a little while, you can view the usage statistics for the hard limit set on pods. If required, list the quotas defined in the project to see the names of all defined quotas:
++
+====
+----
+$ oc get quota -n awesomeproject
+NAME                AGE
+resource-quota      39m
+----
+====
++
+. Describe the resource quota for which you want statistics:
++
+====
+----
+$ oc describe quota resource-quota -n awesomeproject
+Name:			resource-quota
+Namespace:		awesomeproject
+Resource		Used	Hard
+--------		----	----
+pods     		3	10
+----
+====
++
+. Optionally, you can
+link:../admin_guide/quota.html#configuring-quota-sync-period[configure the quota synchronization period], which controls how long to wait before restoring quota usage after resources are deleted.
+. If you want to remove an active quota to no longer enforce the limits of a project:
++
+----
+$ oc delete quota <quota_name>
+----
+
+*Now let's walk through that process and examine all the available options:*
+
+. To create project limits or quotas, you must first write a .yaml file that describes the resource that you want to restrict:
+
+* This *_LimitRange.yaml_* example explains all the
+link:../dev_guide/limits.html#container-limits[container limits] and
+link:../dev_guide/limits.html#pod-limits[pod limits] that you can place within
+your project:
++
+include::admin_guide/limits.adoc[tag=admin_limits_sample_definition]
++
+* These *_ResourceQuota.yaml_* examples explain all the
+link:../admin_guide/quota.html#sample-resource-quota-definitions[Object Counts]
+and
+link:../admin_guide/quota.html#sample-resource-quota-definitions[Compute Resources] that you can place within your project:
++
+include::admin_guide/quota.adoc[tag=admin_quota_object_counts]
++
+include::admin_guide/quota.adoc[tag=admin_quota_compute_resources]
++
+. Once you have defined your limit range object file, you can create a limit range:
++
+----
+$ oc create -f <limit_range_file> -n <project_name>
+----
++
+The `-n` flag is optional. Use it to create the limit in a different project
+than the one you currently are in.
+
+[[limit-number-projects]]
+=== Limiting the Number of Projects a User Can Have
+
+You can
+link:../admin_guide/managing_projects.html#limit-projects-per-user[limit the number of projects] that a user may request by categorizing users with label selectors with the `oc label` command. A label selector consists of the label name and the label value:
+
+----
+label=value
+----
+
+Once users are labeled, you must
+link:../admin_guide/managing_projects.html#modifying-the-template-for-new-projects[modify the default project template] in the *_master-config.yaml_* file
+link:../install_config/configuring_admission_control.html[using an admission control plug-in]. This allows some users to create more projects than others, and you can define different values (or levels) for each label.
```

Comment:
The admission control plug-in content had to change around at the last minute due to some product changes. But if you want to link to something that explains what an admission control plug-in is, there's a small Arch topic now:

../architecture/additional_concepts/admission_controllers.html